cddl/raw-value-compare.cddl

```diff
@@ -0,0 +1,4 @@
+raw-value-compare = [
```

Comment:
There should be a tag before the array. I will add this if the direction is agreed.

Reply:
Agreed.